rustls/tests/api.rs

```diff
@@ -5222,7 +5222,7 @@ fn test_client_sends_helloretryrequest() {
         let wrlen = server.write_tls(&mut pipe).unwrap();
         assert!(wrlen > 200);
         assert_eq!(pipe.writevs.len(), 1);
-        assert!(pipe.writevs[0].len() == 5); // server hello / encrypted exts / cert / cert-verify / finished
+        assert_eq!(pipe.writevs[0].len(), 2); // { server hello / encrypted exts / cert / cert-verify} / finished
```

Comment:
ultra nit: space in the comment before closing `}` for consistency with opening `// { server ..`